Sts1CobcSw/Hal/Spi.hpp

```diff
@@ -10,6 +10,25 @@
 
 namespace sts1cobcsw::hal
 {
+class Spi
+{
+private:
+    RODOS::HAL_SPI spi;
+    RODOS::CommBuffer<std::int64_t> transferEnd;
+
+public:
+    friend auto Initialize(Spi * spiClass, std::int32_t baudRate) -> void;
+
+    template<typename T, std::size_t extent>
+    friend auto ReadFrom(Spi * spiClass, std::span<T const, extent> data, std::int64_t timeout)
```

Comment:
Reading from the SPI requires a span with mutable elements, so remove the `const` after `T`.

This should actually not compile since in the implementation `read()` requires a mutable pointer. The only reason it does is since this is a function template that never gets instantiated in the current code.